test/system/520-checkpoint.bats

```diff
@@ -8,6 +8,10 @@ load helpers.network
 
 CHECKED_ROOTLESS=
 function setup() {
+    # Often breaks on RHEL/CentOS Stream and Fedora Rawhide
+    skip_if_rhel_or_centos
+    skip_if_rawhide
```

Comment:
This is fundamentally incorrect in my eyes.

First this should be tested on RHEL/centos in general, if this breaks there than this is highly concerning and there must be clear bug reports linked *why* it is skipped so QE people can unskip it once the issue is fixed. Otherwise we never catch new issues there which means they won't get fixed either.

As for rawhide there is only a single test case that fails so it should only skip that specific one.